libr/core/cmd_eval.c

```diff
@@ -62,7 +62,7 @@ static bool load_theme(RCore *core, const char *path) {
 	if (!r_file_exists (path)) {
 		return false;
 	}
-	core->cmdfilter = "ec ";
+	core->cmdfilter = "e";
```

Comment:
nope. themes cant change user settings. only colors

Reply:
Then `e scr.color = 3` must set by user manually?.

Reply:
@d4em0n yes.

Reply:
I'm not sure if user uses onedark, the user must know to set scr.color=3